python/mxnet/ndarray_doc.py

```diff
@@ -97,6 +97,40 @@ class CustomDoc(NDArrayDoc):
     >>> output = mx.symbol.Custom(op_type='my_custom_operator', data=input)
     """
 
+class GridGeneratorDoc(NDArrayDoc):
+    """
+    Examples
+    --------
+    >>> data =  mx.nd.array(np.ones((1,2,4,4)) * 1)
+    >>> res = mx.ndarray.GridGenerator(data, "warp")
+    >>> print(res.asnumpy())
+    [[[[-0.33333331  0.33333337  1.          1.66666675]
```

Comment:
These looks ugly. 0.5 steps would look better. It's also too big.

Plus this should go into cpp docs since its not specific to python. Use the cpp doc formatting.